cli/remotebazel/remotebazel.go

```diff
@@ -1042,6 +1043,11 @@ func attemptRun(ctx context.Context, bbClient bbspb.BuildBuddyServiceClient, exe
 		return nil, nil, err
 	}
 	iid := rsp.GetInvocationId()
+	if *invocationIDFile != "" {
+		if err := os.WriteFile(*invocationIDFile, []byte(iid), 0644); err != nil {
```

Comment:
If there are multiple retries, this could get written multiple times. Should we write after all attempts are complete?